### PR TITLE
Issue #68 remove gunicorn

### LINKED ISSUE #68: Feature: remove gunicorn dependency
Log messages get duplicated:

```shell
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:29,889 - gunicorn.error - Starting gunicorn 20.1.0
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:29,889 - gunicorn.error - Listening at: http://0.0.0.0:8000 (11)
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:29,889 - gunicorn.error - Using worker: utils.restartable_worker.RestartableUvicornWorker
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:29,891 - gunicorn.error - Booting worker with pid: 13
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:29,911 - gunicorn.error - Booting worker with pid: 14
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:30,056 - uvicorn.error - Application startup complete.
starlite-example-pg-docker-app-1    | INFO - 2022-09-01 16:31:30,058 - uvicorn.error - Application startup complete.
```

### PR DESCRIPTION
**Pull request by peterschutt**
Closes #68 

Removes gunicorn multi-worker pattern in favor of uvicorn single worker.

### CODE REVIEW

**Comment by peterschutt** [comment]
@grillazz feel free to review, cheers!

File: app/main.py [hunk 1]
```diff
@@ -21,4 +21,11 @@
 )
 
 if __name__ == "__main__":
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    uvicorn.run(
```
Comment (grillazz) [inline app/main.py:24]: i saw you added uvloop - not sure as per loop=auto setting uvloop will be set

**Comment by Goldziher** [comment]
you can kill the restartable worker as well, no?

**Comment by peterschutt** [comment]
> you can kill the restartable worker as well, no?

I have removed it - what am I missing? There's a coverage omit directive in pyproject left dangling.. I"ll get that now..

File: app/main.py (hunk 1 above)
Comment (peterschutt) [reply app/main.py:24]: See https://github.com/encode/uvicorn/blob/master/uvicorn/loops/auto.py

**Pull request merged**